### PR TITLE
Feature/support for admin modifiable sidefields/helptexts

### PR DESCRIPTION
**Pull request by pelmat**
# Feature/support for admin modifiable sidefields/helptexts

## Added backend support for sidefields that are modifiable through django admin for easier access.

-----------------------------------------------------------------------------------------------
#### Changes:
 1. events/admin.py
     * Added support for sidefields in django admin. Made visually appealing.

2. events/api.py
     * Added serializer, viewset and query params for sidefields.

3. events/importer/base.py
     * Fixed and added support to skip the required language and datasource attributes for importer classes. Does not affect existing importers. Documented in the base importer, example in use; helptexts.py importer.

4. events/importer/helptexts.py
     * Added importer for default sidefield helptexts. Re-run prevented. Importer follows the new instantiation base configuration modifications.

5. events/migrations/0087_sidefield.py
     * Added migration file for sidefields.

6. events/models.py
     * Added new model for sidefields.

7. events/translation.py
    * Added multilanguage support to sidefields.

### CODE REVIEW

File: events/importer/base.py [hunk 1]
```diff
@@ -60,11 +60,22 @@ def __init__(self, options):
...
-        self.setup()
+        # Pre processing phase runs your importers setup.
+        # You can override this check by setting 'ignore_setup' = True into your importer.
 
-        # this has to be run after setup, as it relies on organization and data source being set
-        self._images = {obj.url: obj for obj in Image.objects.filter(publisher=self.organization,
-                                                                     data_source=self.data_source)}
+        ignore_setup_phase = getattr(self, 'ignore_setup', False)
+
+        if ignore_setup_phase == False:
```
Comment (ezkat) [inline events/importer/base.py]: [Don’t compare boolean values to True or False using ==](https://peps.python.org/pep-0008/)

File: events/importer/base.py [hunk 2]
```diff
@@ -60,11 +60,22 @@ def __init__(self, options):
...
+            self.setup()
+
+        # Post processing phase checks if importers have data_source & organization attributes set.
+        # You can override this check by setting 'ignore_prerequisites' = True into your importer.
+
+        prerequisites = getattr(self, 'ignore_prerequisites', False)
+
+        if prerequisites == False:
```
Comment (ezkat) [inline events/importer/base.py]: [Don’t compare boolean values to True or False using ==](https://peps.python.org/pep-0008/)

Comment (ghost) [reply events/importer/base.py]: My bad, will fix.

**Comment by Anthon98** [comment]
For some reason the latest fix commit isn't showing.

File: events/importer/base.py (hunk 1 above)
Comment (ghost) [reply events/importer/base.py]: Should be fixed now.

File: events/importer/base.py (hunk 2 above)
Comment (ghost) [reply events/importer/base.py]: Should be fixed now.

**Review by ezkat — APPROVED** [review]

**Pull request merged**